kubetest/main.go

```diff
@@ -40,13 +40,12 @@ import (
 const defaultGinkgoParallel = 25
 
 var (
-	artifacts    = initPath("./_artifacts")
-	interrupt    = time.NewTimer(time.Duration(0)) // interrupt testing at this time.
-	kubetestPath = initPath(os.Args[0])
```

Comment:
This need to be `$WORKSPACE/kubetest`, instead of `cwd/kubetest`

Reply:
and now it's wrong as well - we change dir quite often within kubetest and this will break other jobs.